docs/DeveloperGuide.md

```diff
@@ -736,72 +750,110 @@ Similar to <u>deleting an elderly (UC3)</u> but only deleting an elderly's Nok d
     * 2c1. NurseyBook shows an error message.
 
       Use case resumes at step 1.
-    
-**UC9: View an elderly's details**
+* *a. At any time, user requests to <u>view help (<a href="#uc19-viewing-help">UC19</a>)</u>.
+
+##### UC10: List elderly with queried tags
 
 **MSS**
 
-1. User requests to <u>list elderly (UC1)</u>
-2. User requests to view details of a specific elderly in the list based on the index.
-3. NurseyBook shows details of selected elderly and command success message.
+1. User requests to filter elderly based on queried tags.
+2. NurseyBook shows a list of elderly that have all the tags queried.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
-* 2a. The given index is invalid.
+* 1a. The given tag is invalid.
+    * 1a1. NurseyBook shows an error message.
+
+      Use case resumes at step 1.
+* 1b. There are extra parameters not accepted by the command.
+    * 1b1. NurseyBook shows an error message.
+
+      Use case resumes at step 1.
+* 2a. The list of elderly is empty. 
+
+  Use case ends.
+* *a. At any time, user requests to <u>view help (<a href="#uc19-viewing-help">UC19</a>)</u>.
+
+##### UC11: Add remark about an elderly
+
+**MSS**
+
+1. User requests to <u>list elderly (<a href="#uc1-list-elderly">UC1</a>)</u>.
+2. User requests to add a remark to a specific elderly in the list based on index.
+3. NurseyBook adds the remark to the elderly.
+4. NurseyBook shows updated list of elderly and command success message.
+
+   Use case ends.
+
+**Extensions**
+* 2a. The given parameter is invalid.
     * 2a1. NurseyBook shows an error message.
 
       Use case resumes at step 1.
+* 2b. There are extra parameters not accepted by the command.
+    * 2b1. NurseyBook shows an error message.
+
+      Use case resumes at step 1.
+* 2c. The edit does not cause any change in elderly's remark.
+    * 2c1. NurseyBook shows an error message.
 
-**Use cases of task commands**
+      Use case resumes at step 1.
+* *a. At any time, user requests to <u>view help (<a href="#uc19-viewing-help">UC19</a>)</u>.
+
+#### Use cases of task commands
 
-**UC10: List tasks**
+##### UC12: List tasks
 1. User requests to list tasks.
 2. NurseyBook shows a list of tasks.
    
-    Use case ends.
+   Use case ends.
 
 **Extensions**
-* 1a. User requests to find tasks with matching keywords
+* 1a. User requests to <u>find tasks with matching keywords(<a href="#uc17-find-a-task">UC17</a>)</u>.
     * 1a1. NurseyBook shows a list of tasks that matches the user's query (by description)
 
       Use case ends.
 
```

Comment:
I think this part can be removed after linking to UC17. I can change it in my next PR!